tools/build-addons.mjs

```diff
@@ -0,0 +1,61 @@
+// Usage: e.g. node build-addons.mjs <path to node-gyp> <directory>
```

Comment:
Can we add a hashbang line to clarify that it's supposed to be an entry-point file? Even better if you can `chmod +x tools/build-addons.mjs`.
```suggestion
#!/usr/bin/env node
// Usage: e.g. node build-addons.mjs <path to node-gyp> <directory>
```

Reply:
certainly yes!